README.md

```diff
@@ -1,24 +1,159 @@
+    <li><a href="https://ruby-lang.org/">Ruby</a></li>
+  </ul>
+</details>
 
-* Services (job queues, cache servers, search engines, etc.)
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
 
-* Deployment instructions
+### Key Features <a name="key-features"></a>
 
-* ...
+- Add a new game, a music album and a book
+- Add extra details such genre, label and authors
+- All the data is persisted in JSON files
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- A text editor preferably [Visual Studio code](https://code.visualstudio.com/)
+- Latest version of [Ruby](https://www.ruby-lang.org/en/downloads/) installed
+- [PostgreSQL Server](https://www.postgresql.org/download/)
+
+### Setup
+
+Clone this [repository]()https://github.com/nunegabriel/blog-app.git to your desired folder:
+
+```sh
+  cd my-folder
+  https://github.com/nunegabriel/blog-app
+  cd blog-app
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  gem install
+```
+
+### Usage
+
+To run the project, execute the following command in your terminal:
+
+```sh
+  ruby src/main.rb
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  rspec spec
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Gabriel Nunekpeku**
+
+- GitHub: [@nunegabriel](https://github.com/Nowembabazi)
+- Twitter: [@twitterhandle](https://twitter.com/_cornrow)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/gabriel-nunekpeku-623608173/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] A graphical user interface
+- [ ] Better information validation
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, please leave a star 😁
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+We would like to thank the Microverse staff for all the efforts they put in define the requirements of this project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+- **Can I reuse this code?**
+
+  - Yes, feel free to fork it
+
+- **Do I need knowledge of Ruby to use this project?**
+
+  - No you do not.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] You have a link here to your license file but you didn't add any license file in your repository :fearful: Kindly add a license file :+1: